app/buck2_core/src/fs/fs_util.rs

```diff
@@ -334,6 +335,17 @@ pub fn remove_all<P: AsRef<Path>>(path: P) -> anyhow::Result<()> {
     r
 }
 
+pub fn read_to_bytes<P: AsRef<Path>>(path: P) -> anyhow::Result<Vec<u8>> {
```

Comment:
You might want to do `let path = P::as_ref(&path)` or similar on the first line, and then avoid repeating it below. But your choice.